packages/material-ui/src/SvgIcon/SvgIcon.js

```diff
@@ -32,6 +31,12 @@ export const styles = theme => ({
   colorDisabled: {
     color: theme.palette.action.disabled,
   },
+  fontSizeDefault: {
```

Comment:
it probably makes more sense to set this inside root and remove the whole `fontSizeDefault` class, does it?

Reply:
I agree 👍